### DISCUSSION TITLE
Calculating with handlebars

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#6799] [Help] [answered]
Hey,

I have a little two liner that simply calculates 3 values, is there a way to add all 3 values together?

`{{ times Form.Fields.Item#1price Form.Fields.Item#1amount times Form.Fields.Item#2price Form.Fields.Item#2amount times Form.Fields.Item#3price Form.Fields.Item#3amount }}`

Thanks!

**Comment by R2bEEaton** [comment, marked as answer]
Hi, sorry if I'm misunderstanding your question. It seems you want to add the following three values together:

```
{{ times Form.Fields.Item#1price Form.Fields.Item#1amount }}
{{ times Form.Fields.Item#2price Form.Fields.Item#2amount }}
{{ times Form.Fields.Item#3price Form.Fields.Item#3amount }}
```

If this is the case, this can be achieved using parentheses and the `sum` helper:
```
{{ sum ( times Form.Fields.Item#1price Form.Fields.Item#1amount ) ( times Form.Fields.Item#2price Form.Fields.Item#2amount ) ( times Form.Fields.Item#3price Form.Fields.Item#3amount ) }}
```

You may have been using the `add` helper, but it only takes two parameters; `sum` takes a whole list.
Hope this is what you're looking for!

**Reply by kasgithub8** [reply]
Hi @R2bEEaton,

it doesn't seem to work. Do I need a extra column in my table for that?

**Reply by R2bEEaton** [reply]
I may not have understood what you were asking... If you want this sum in another column, you can add a "formula" column and set the formula to that handlebar I sent.

If you want to display the sum on a page, just use the handlebar expression.

Can you explain further what your use case is? Regardless, the helper for adding values together is `sum` so definitely use that when you want to add them.

**Reply by kasgithub8** [reply]
@R2bEEaton 
It would look like this: 
![image](https://user-images.githubusercontent.com/95284278/181179756-192c44a2-fd1e-4ebb-8dfe-b84aef8de309.png)

If this is correct, do I have to add a field to the form screen? Because I want to display the "sumTest" in a table.

Thanks!

**Reply by kasgithub8** [reply]
I got it now, thanks tho
